src/browser/modules/D3Visualization/components/DetailsPane.tsx

```diff
@@ -39,17 +40,54 @@ import { StyleableRelType } from './StyleableRelType'
 import { upperFirst } from 'services/utils'
 import { ShowMoreOrAll } from 'browser-components/ShowMoreOrAll/ShowMoreOrAll'
 
+export const MAX_LENGTH_SMALL = 150
+export const MAX_LENGTH_LARGE = 300
+type ExpandableValueProps = {
+  value: string
+  type: string
+  wideMode: boolean
+}
+function ExpandableValue({ value, wideMode }: ExpandableValueProps) {
+  const [expanded, setExpanded] = useState(false)
+
+  const maxLength = wideMode ? MAX_LENGTH_LARGE : MAX_LENGTH_SMALL
```

Comment:
I think it's nice to have the max string length be dependant on how much space is available, but I was thinking it'd be easier & simpler if we calculated some maxLength from the width? So rather than having a wideMode prop we could send just the "width" and perhaps say maxLenght = width/3 or something like that? Would get rid of a few constants and be more flexible. WDYT?

Reply:
For a followup PR